### PR TITLE
Mirid peripheral override code snippet updates

### PR DESCRIPTION
**Pull request by adrianstevens**

### CODE REVIEW

File: docfx/api-override/Meadow.Foundation.Grove/Meadow.Foundation.Grove.Sensors.Switches.SwitchP.md [hunk 1]
```diff
@@ -3,10 +3,10 @@ uid: Meadow.Foundation.Grove.Sensors.Switches.SwitchP
 remarks: *content
 ---
 
-| Switch(P) | |
+| SwitchP | |
```
Comment (patridge) [inline docfx/api-override/Meadow.Foundation.Grove/Meadow.Foundation.Grove.Sensors.Switches.SwitchP.md:6]: Did we need a hack for the real name on all these component variations ("Switch(P)")?

File: docfx/api-override/Meadow.Foundation/Meadow.Foundation.Sensors.Rotary.RotaryEncoder.md [hunk 2]
```diff
@@ -51,23 +51,13 @@ public override Task Initialize()
     //==== Classic Events
     rotaryEncoder.Rotated += RotaryEncoder_Rotated;
 
-    //==== IObservable
```
Comment (patridge) [inline docfx/api-override/Meadow.Foundation/Meadow.Foundation.Sensors.Rotary.RotaryEncoder.md:54]: Did we intend to remove the Observable example here and in the next example?

**Review by patridge — APPROVED** [review]
Looks good. Had a few non-blocking questions.

File: docfx/api-override/Meadow.Foundation/Meadow.Foundation.Sensors.Rotary.RotaryEncoder.md (hunk 2 above)
Comment (adrianstevens) [reply docfx/api-override/Meadow.Foundation/Meadow.Foundation.Sensors.Rotary.RotaryEncoder.md:54]: yes - and thx for checking

**Pull request merged**